openshift/ocp_on_openstack.adoc

```diff
@@ -89,6 +89,123 @@ Push the `pass` changes to the git repository.
 git push
 ----
 
+
+== Post installation steps
+
+
+=== Install jump server
+
+To be able to SSH to any of the OCP cluster machines a _jump server_ is required.
+
+==== Requirements
+
+1. Obtain the name of the network interface created by the OCP installation process
+  * As a result of the OCP installation process on RHOS a network is created that allows communication between the cluster instances. 
+  * The name should resemble `ocp-<code>-openshift` where `ocp` is the prefix our ansible role applies to the installation and `<code>` is a dynamic string generated by the installation process.
+
+[NOTE]
+====
+For the sake of example the `ocp-sscpc` prefix will be used.
+====
+
+==== Create and configure the server
+
+[WARNING]
+====
+One _jump server_ will be required so the OCP cluster name prefix should be used when creating this VM.
+====
+
+[source,bash]
+----
+ansible-playbook ansible/playbook/openstack/openstack_vm_create_passwordstore.yml \
+  -e '{"openstack": {"vm": {"network": "ocp-sscpc-openshift","image": "Fedora-Cloud-Base-29", "flavor": "m1.small"}}}' \
+  -e vm_name=ocp-sscpc-jump-server
+----
+
+Create a _Floating IP_ from the external network that one's router is connected to, in our case `provider_net_cci_13`. As the _jump server_ is created on the same network as the OCP cluster it isn't accesible from the outside. To access this server a RHOS floating IP is required. The `provider_net_cci_13` is the team's public RHOS network name.
```

Comment:
Can you explain how you get such an external network ? what is the router you refers here ?

Reply:
Improved the docs to better explain this paragraph.